scripts/sendpayment.py

```diff
@@ -81,8 +81,13 @@ def main():
             destaddr = args[2]
         mixdepth = options.mixdepth
         addr_valid, errormsg = validate_address(destaddr)
-        if not addr_valid:
+        command_to_burn = (is_burn_destination(destaddr) and sweeping and
```

Comment:
Chuckle, this is a very good example of what I meant [here](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/536#issuecomment-624633575) :) (not that I suggest we try to address it in this PR, it will have to be done later).

Reply:
Completely agreed with this. Feature creep is a thing.